src/Node/Expression/Binary/EqualBinary.php

```diff
@@ -15,6 +15,21 @@
 
 class EqualBinary extends AbstractBinary
 {
+    public function compile(Compiler $compiler)
+    {
+        if (version_compare(PHP_VERSION, '8.0.0', '>=')) {
```

Comment:
should use `PHP_VERSION_ID` for consistency

Reply:
done